packages/snjs/lib/services/api/session_manager.ts

```diff
@@ -120,6 +120,11 @@ export class SNSessionManager extends PureService<SessionEvent> {
     }
   }
 
+  public getPasswordCreatedDate(): Date | undefined {
```

Comment:
Sorry didn't catch this earlier, but I would put this function in protocol_service instead since root key is the domain of the protocol service.

Reply:
Good point, yeah, `getRootKey` comes from protocol service, so it makes more sense, done.